### DISCUSSION TITLE
Firebase & APP_MEASUREMENT_ID="" does not exist in our firebaseConfig

### DISCUSSION THREAD

**Discussion by jimguistical** [nss-evening-cohort-14/evening-client-side#192] [Help Me!] [answered]
Jim Conner
Have you searched the other help tickets? Yes

Describe your issue: Our firebaseConfig file does not have `APP_MEASUREMENT_ID=""`, so we can't add it to our local .env file.  We are not sure if this would effect our app going forward or not.  We confirmed that other repos, Almost Amazon for example, did contain the MEASURMENT_ID. 

What have you googled?
https://stackoverflow.com/questions/59784731/firebaseerror-analytics-measurementid-field-is-empty-in-firebase-config
https://stackoverflow.com/questions/60773112/app-web-measurement-id-and-the-collect-endpoint
https://stackoverflow.com/questions/60804074/how-to-get-the-measurementid-from-the-firebase-config

https://github.com/nss-evening-cohort-14/whalom-park-dashboard-whalom-park-dashboard

screenshot from firebase -- highlighted area where Measurment_ID should be. 

![Screenshot 2021-03-16 193232](https://user-images.githubusercontent.com/76793747/111397354-1c68af00-868f-11eb-8dc3-e6eace64de81.jpg)

**Comment by ajawashington** [comment, marked as answer]
@jim-conner it would be in your CDN and not config. But also you do not need that for your project.

**Reply by jimguistical** [reply]
Okay, sounds good -- thank you, Aja!
